website/content/apis/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: API reference docs
 description: MetalLB API reference documentation
+weight: 10
```

Comment:
This file is generated automatically.

Reply:
Removed.
I mistakenly thought it was necessary for the order of the links.